plugins/CoreHome/tests/Integration/LoginAllowlistTest.php

```diff
@@ -135,6 +135,12 @@ public function test_getAllowlistedLoginIps_shouldResolveIp()
         $this->assertSame(['192.168.33.1', '185.31.40.177', '127.0.0.1'], $this->allowlist->getAllowlistedLoginIps());
     }
 
+    public function test_getAllowlistedLoginIps_shouldResolveIpv6()
+    {
+        $this->setGeneralConfig('login_allowlist_ip', ['192.168.33.1', 'integration-test.matomo.org', '127.0.0.1']);
```

Comment:
added this DNS entry specifically for this test

Reply:
@Findus23 @sgiehl would it be an issue that `https://integration-test.matomo.org/` resolves `aaaa` to `::1`?

Reply:
can't think of any reason why that could be an issue